assets/src/scripts/components/dailyValueHydrograph/selectors/time-series-lines.js

```diff
@@ -0,0 +1,66 @@
+import {DateTime} from 'luxon';
+import isEqual from 'lodash/isEqual';
+import {createSelector} from 'reselect';
+
+import {getCurrentObservationsTimeSeries} from '../../../selectors/observations-selector';
+
+const TWO_DAYS = 1000 * 60 * 60 * 24 * 2;
```

Comment:
Might want to add a comment here about units (i.e. milliseconds in 2 days).

Reply:
Will do